core/src/main/java/com/google/errorprone/bugpatterns/SizeGreaterThanOrEqualsZero.java

```diff
@@ -242,40 +238,48 @@ private Description provideReplacementForStaticMethodInvocation(
       MethodInvocationTree callToSize,
       final VisitorState state,
       ExpressionType expressionType) {
-    ExpressionTree classToken = ASTHelpers.getReceiver(callToSize);
+    ExpressionTree classToken = getReceiver(callToSize);
 
     if (HAS_EMPTY_METHOD.matches(classToken, state)) {
       List<CharSequence> argumentSourceValues =
-          Lists.transform(
-              callToSize.getArguments(),
-              new Function<ExpressionTree, CharSequence>() {
-                @Override
-                public CharSequence apply(ExpressionTree expressionTree) {
-                  return state.getSourceForNode((JCTree) expressionTree);
-                }
-              });
+          callToSize
+              .getArguments()
+              .stream()
+              .map(state::getSourceForNode)
+              .collect(toImmutableList());
       String argumentString = Joiner.on(',').join(argumentSourceValues);
```

Comment:
@epmjohnston I believe that this portion of the code could be shortened and made a bit more efficient with something like:

```java
String argumentString =
  callToSize
    .getArguments()
    .stream()
    .map(state::getSourceForNode)
    .collect(joining(","));
```